wcfsetup/install/files/ts/WoltLabSuite/Core/Bootstrap.js

```diff
@@ -69,9 +69,7 @@ define(
         DateTimeRelative.setup();
         DatePicker.init();
         UiSimpleDropdown.setup();
-        UiMobile.setup({
-          enableMobileMenu: options.enableMobileMenu
-        });
+        UiMobile.setup(options.enableMobileMenu);
```

Comment:
Isn't this a BC break?

Reply:
Yes, but this is an internal module not referenced anywhere else.